take-your-first-steps-with-rust/module-3/includes/6-knowledge-check.md

```diff
@@ -0,0 +1,19 @@
+# Knowledge Check
```

Comment:
@tilacog I'm in progress of reviewing this module.

One ask I have going forward is if you can provide a short explanation for each answer describing why it's either correct or inccorect? Let me know if that's okay with you, I know it wasn't something I had asked for previously.

Reply:
Sure, no problem.
I'm just wondering how should I organize those explanations in the markdown file. 
Could you give me a brief example of how a question would be written, please?

Reply:
Good question! You could put the explanations after each answer. Feel free to add it on as another bullet, for example:

- Access an out-of-bounds index of a Vector with `vector[index]` notation
  - This notation will result in a panic error since the index of the vector is not being accessed correctly.
- Access an out-of-bounds index of a Vector with `vector.get(index)` notation
  - Correct. This is the right way to get the index of a vector.